domain-management/src/main/java/org/jboss/as/domain/management/security/adduser/AddUser.java

```diff
@@ -405,6 +413,12 @@ public String instructions() {
             public String instructions() {
                 return DomainManagementLogger.ROOT_LOGGER.argHelp();
             }
+        },
+        DISPLAY_SECRET("-sv", "--secret") {
```

Comment:
IMO I think this may be better to be something along the lines of --display-secret and -ds this way the abbreviated form would have a closer relationship with the full form - also there will be no ambiguity of anyone thinks the argument means to set the secret value.